conans/client/conf/__init__.py

```diff
@@ -99,6 +99,14 @@ def proxies(self):
         except:
             return None
 
+    def settings_defaults_set(self, settings):
```

Comment:
It is very used in testing, it would require to change quite a few tests, so I tried to keep the PR as short as possible. I can fix it if you think it is worth.